array/DNAm/preprocessing/installLibraries.r

```diff
@@ -11,11 +14,21 @@ if (!require("BiocManager", quietly = TRUE))
 BiocManager::install("bigmelon")
 
 #GDS script
-BiocManager::install("IlluminaHumanMethylationEPICanno.ilm10b4.hg19") #This one is commented out in original script so unsure if necessary
-BiocManager::install("IlluminaHumanMethylationEPICanno.ilm10b2.hg19")
-BiocManager::install("IlluminaHumanMethylationEPICmanifest")
-BiocManager::install("IlluminaHumanMethylation450kanno.ilmn12.hg19")
-BiocManager::install("IlluminaHumanMethylation450kmanifest")
+
+if(arrayType=='450K'){
+  #library(IlluminaHumanMethylationEPICanno.ilm10b4.hg19)
+  BiocManager::install(IlluminaHumanMethylation450kanno.ilmn12.hg19)
+  BiocManager::install(IlluminaHumanMethylation450kmanifest)
+}
+if(arrayType=='EPICv1'){
+  BiocManager::install(IlluminaHumanMethylationEPICanno.ilm10b2.hg19)
+  BiocManager::install(IlluminaHumanMethylationEPICmanifest)
+}
+if(arrayType=='EPICv2'){
+	install("jokergoo/IlluminaHumanMethylationEPICv2manifest")

```

Comment:
I think there are now versions of these on bioconductor - do we want to change to those for consistency?